src/http/index.test.ts

```diff
@@ -1,3 +1,4 @@
+import { describe, expect, it } from '@jest/globals';
```

Comment:
Alternatively we could add the rule `'jest/prefer-importing-jest-globals': 'off',` either here, or even upstream to `@api3/eslint-plugin-commons`

Reply:
Yeah, I'm fine with upstreaming. This would be quite annoying to do in the existing repos.

Reply:
Can you make a patch and release 1.0.1?

Reply:
Yes - https://github.com/api3dao/eslint-plugin-commons/issues/10 and https://github.com/api3dao/eslint-plugin-commons/issues/11

Reply:
Done